src/agenteval/cli.py

```diff
@@ -17,11 +20,18 @@
     upload_folder_to_hf,
     upload_summary_to_hf,
 )
-from .models import EvalConfig, EvalResult
+from .models import EvalConfig, TaskResults
+from .leaderboard.models import LeaderboardSubmission
 from .score import process_eval_logs
 from .summary import compute_summary_statistics
-
-EVAL_FILENAME = "agenteval.json"
+from .io import atomic_write_file
+
+HF_URL_PATTERN = r"^hf://([^/]+/[^/]+)/(.*)$"
```

Comment:
nit: 
```suggestion
HF_URL_PATTERN = r"^hf://(?P<repo_id>[^/]+/[^/]+)/(?P<path>.*)$"
```
since the capture groups are actually being used